pkg/sshutil/sshutil.go

```diff
@@ -138,8 +138,10 @@ func CommonOpts(useDotSSH bool) ([]string, error) {
 	var opts []string
 	if runtime.GOOS == "windows" {
 		privateKeyPath = ioutilx.CanonicalWindowsPath(privateKeyPath)
+		opts = []string{fmt.Sprintf(`IdentityFile='%s'`, privateKeyPath)}
+	} else {
+		opts = []string{fmt.Sprintf(`IdentityFile="%s"`, privateKeyPath)}
 	}
-	opts = []string{fmt.Sprintf(`IdentityFile="%s"`, privateKeyPath)}
```

Comment:
Wondering if we can use the "windows" form for non-windows too

Reply:
I tried that at first, but some of the e2e tests were failing :(. Example: https://github.com/lima-vm/lima/actions/runs/6355908876